### ISSUE TITLE
Problems syncing to Garmin

### ISSUE THREAD

**Issue by toLLto** [cpfair/tapiriik#526]
I tried several times to setup sync from Strava to Garmin but it's not working. From tapiriik.com I can successfully connect to Garmin. But the strange thing is - no user "tapiriik-sync-kedgwick" is added to my Garmin account, to my knowledge this should be done automatically. 
Next point is - I've enabled Data Upload in Garmin privacy settings ("grant Garmin Connect permission to accept data uploads") but still see the message to reauthorize Garmin connection. 
I have severall time reconnected Garmin Account and pushed the "Reauthorize" button - nothing change. My Strava activities still won't sync to Garmin. I see only this error message: "Garmin Connect: There was an error while synchronizing this activity."

**Comment by RobDickson** [comment]
It's no longer possible to upload activities to Garmin using its new API.

**Comment by mb-aero** [comment]
Any news on this issue? What's the workaround any of you are using?
Thanks

**Comment by mb-aero** [comment]
Yes, it still doesn’t work. though I wouldn’t call it broken because Garmin intends It to not be able to upload Data
It’s a shame, because the files that I have are TCX and all sport types and title names are lost when I manually upload them to Garmin connect. with the old tapiriik working it always put the title in correctly.
